src/App.tsx

```diff
@@ -24,25 +23,26 @@ const Main = lazy(() => import('pages/Main'))
 const EmailProof = lazy(() => import('pages/EmailProof'))
 
 const pageContainer = classnames(
-  space('space-y-6', 'sm:space-y-0'),
   display('flex'),
   flexDirection('flex-col'),
-  minHeight('min-h-screen'),
-  justifyContent('justify-between')
+  minHeight('min-h-screen')
+)
+
+const bodyContainer = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  margin('my-6', 'md:mt-0', 'md:mb-auto', 'mx-auto')
```

Comment:
Let's move `mx-auto` to the left part of this, because you have left part for `mobiles` and right part for `desktops`

Reply:
Did it